<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsApplication2Class</class>
 <widget class="QMainWindow" name="QtWidgetsApplication2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsApplication2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_sex">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>90</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>性别:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_age">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>年龄</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_address">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>住址</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QRadioButton" name="rbn_male">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>男</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbn_female">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>女</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_age">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_address">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbn_add">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbn_del">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>250</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbn_alter">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>250</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>614</x>
      <y>290</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>删除某一单元格</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWidgetsApplication2.qrc"/>
 </resources>
 <connections/>
</ui>
